### ISSUE TITLE
<realm/util/features.h> not found

### ISSUE THREAD

**Issue by gkaimakas** [realm/realm-cocoa#3092]
I updated to RealmSwift 0.97.0 and since that I am getting the error that <realm/util/features.h> cannot be found.

I tried going back to version 0.96.1 but the same error occurs.

I am using cocoapods 0.39.

**Comment by kwosu87** [comment]
I have a same issue. I'm using cocoapods 1.0.0-beta.2

**Comment by gkaimakas** [comment]
I had upgraded to version 1.0.0-beta.2 of cocoapods and then reverted back to 0.39.0 by uninstalling version 1.0.0-beta.2.

Apparently some of the plugins of version 1.0.0-beta.2 remain, such as cocoapods-try, cocoapods-stats etc.

I was able to fix the issue following these steps:

1) Uninstall every version of cocoapods, including plugins
2) Install version 0.39.0
3) Delete podfile.lock, pods folder and xcworkspace from your project directory.
4) Run pod install again

Probably some weird bug on cocoapods and not on realm
